docs/content/latest/tools/arctype.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Use Arctype with YugabyteDB YSQL
+headerTitle: Arctype
+linkTitle: Arctype
+description: Use Arctype to work with distributed SQL databases in YugabyteDB.
+menu:
+  latest:
+    identifier: arctype
+    parent: tools
+    weight: 2710
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to query and visualize data in YugabyteDB using [Arctype](https://arctype.com/), a user-friendly collaborative SQL client.
+
+Arctype is a database client with a focus on speed and design. Arctype is free to use and cross platform. It offers one click query sharing for teams, and users can quickly visualize query output and even combine multiple charts and tables into a simple and intuitive dashboard.
+
+![Arctype application](/images/develop/tools/arctype/Arctype-YB-Image-2.png)
+
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to install the Arctype client on your computer. You can download clients are available for Windows, Linux, and Mac from the [Arctype](https://arctype.com/) website.
+
+## Create a database connection
+
+Follow these steps to connect your Arctype desktop client to YugabyteDB:
+
+1. Launch the Arctype desktop client.
+
+1. Follow the in-app prompts to create and log into your Arctype account.
+
+1. On the "Connect a Database" step, select PostgreSQL, and click Continue.
+
+    ![Connect DB Step](/images/develop/tools/arctype/arctype-conect_step3.png)
+
+1. Enter your YugabyteDB host and port information, and click Continue.
+
+    ![Enter host and port](/images/develop/tools/arctype/arctype-connect-step4.png)
+
+1. Enter your database name, username, password, and nickname for this database, and click Continue.
+
+    ![Enter database connection details](/images/develop/tools/arctype/arctype-connect-step5.png)
+
+1. Enter a workspace name, and click Continue.
+
+You've successfully created a connection to your YugabyteDB database, and you can now start querying and visualizing your DB using Arctype.
+
+## What's Next
```

Comment:
As the the last step in "Create a database connection", you need to include an actual test of your connection. See, for example, the last step in the [DBeaver doc](https://docs.yugabyte.com/latest/tools/dbeaver-ysql/). When I get to the end of the steps, I want to know that I've been successful!

Reply:
added test connection step